demo/views/demo-all.dust

```diff
@@ -46,14 +46,11 @@
   <input type="text" class="offleft">
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
It works when I run this locally, but it fails when I deploy it to Bluemix for visual review.. So we need to think of a process where it will work both ways.